bukkit/rpk-unconsciousness-bukkit/src/main/kotlin/com/rpkit/unconsciousness/bukkit/listener/PlayerMoveListener.kt

```diff
@@ -33,9 +33,9 @@ class PlayerMoveListener : Listener {
         val minecraftProfileService = Services[RPKMinecraftProfileService::class.java] ?: return
         val characterService = Services[RPKCharacterService::class.java] ?: return
         val unconsciousnessService = Services[RPKUnconsciousnessService::class.java] ?: return
-        val minecraftProfile = minecraftProfileService.getPreloadedMinecraftProfile(event.player) ?: return
-        val character = characterService.getPreloadedActiveCharacter(minecraftProfile) ?: return
-        if (!unconsciousnessService.isUnconscious(character)) return
+        val minecraftProfile = minecraftProfileService.getMinecraftProfile(event.player) ?: return
+        val character = characterService.getActiveCharacter(minecraftProfile) ?: return
+        if (!unconsciousnessService.isUnconscious(character).join()) return
```

Comment:
PlayerMoveEvent is fired synchronously, don't use join (_especially_ with PlayerMoveEvent, we never ever want to run database queries in something this frequent)
`character` is a CompletableFuture, maybe use preloaded.